jupyterhub/requirements.txt

```diff
@@ -17,6 +17,6 @@
 # limitations under the License.
 
 # Installing from Git until a new version of kubespawner (introducing 'delete_grace_period') is released.
--e git://github.com/jupyterhub/kubespawner.git@46a4b109c5e657a4c3d5bfa8ea4731ec6564ea13#egg=jupyterhub-kubespawner
-jupyterhub==0.9.6
+-e git://github.com/jupyterhub/kubespawner.git@01373369044f8fd9754956bb6a3b3cfeecffb939#egg=jupyterhub-kubespawner
+jupyterhub==1.0.0
```

Comment:
Can't you simply get rid of this line? Doesn't any `jupyterhub/k8s-hub:0.9-dev` image come with jupyterhub 1.0.0 installed?

Reply:
that's true, I can remove it. I am wondering if it makes sense to leave it here anyway to make it clearer that `jupyterhub` must be updated also here -- otherwise it's not that intuitive for someone looking for `jupyterhub` that `jupyterhub/k8s-hub:0.9-dev` requires `jupyterhub` 1.0.0